src/pages-and-resources/calculator/Settings.jsx

```diff
@@ -0,0 +1,46 @@
+import { getConfig } from '@edx/frontend-platform';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import React from 'react';
+import AppSettingsModal, { useAppSettingsForm } from '../app-settings-modal/AppSettingsModal';
+
+import messages from './messages';
+
+const CALCULATOR_HELP_URL = getConfig().CALCULATOR_HELP_URL
+  || 'https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/exercises_tools/calculator.html';
```

Comment:
for now, we are thinking which layer should be responsible for providing this data, we have examples from discussion providers where help links are being provided by the backend API. We will discuss this internally and get back to you on this

Reply:
So, I confirmed with edX team the preferred way is to send these URLs from the backend API responsible for sending the settings back to the frontend.

There is an existing example where (similar) data is served from the backend => see providers.active data in this API.
https://courses.stage.edx.org/discussions/api/v0/course-v1:UBCx+Water201x_2+2T2015

Reply:
I understand that this data will be coming from the backend eventually. However, I have implemented it as a constant here, while that part is being figured out. We can change the source of this message later, but for now we need some value.

Reply:
Okay, so I have created a ticket for that and assigning it to you so we can fix that in a separate ticket. https://openedx.atlassian.net/browse/TNL-8446